app/templates/help.hbs

```diff
@@ -9,26 +9,63 @@
     <div class="subheading align-center" data-test-help-page-navigation-links>
       View our most read
       <a href={{docsUrl}} target="_blank" ref="noopener noreferrer">documentation</a>,
-      popular topics in our
-      <a href={{communityUrl}} target="_blank" ref="noopener noreferrer">Community forum</a>,
-      or
-      <a class="pointer" onclick={{action 'setAnchor' 'form'}}>talk with us!</a>
+      {{#if isPro}}
+        popular topics in our
+        <a href={{communityUrl}} target="_blank" ref="noopener noreferrer">Community forum</a>,
+        or
+        <a class="pointer" onclick={{action 'setAnchor' 'form'}}>talk with us!</a>
+      {{else}}
+        or talk with us in our
+        <a href={{communityUrl}} target="_blank" ref="noopener noreferrer">Community forum</a>!
```

Comment:
This looks like a good candidate for `external-link-to`

Reply:
Good point, agreed :smile: Though I ended up changing these to instead do the `setAnchor` action so that they would scroll to their respective sections.